### DISCUSSION TITLE
How to display all members of an Object, without showing the object itself ?

### DISCUSSION THREAD

**Discussion by astariul** [mkdocstrings/mkdocstrings#635] [Q&A]
I have a module `oblique.api` and here is my `.md` file : 

```markdown
# api.py

## Exception handling

::: oblique.api.MyCustomException
    options:
        heading_level: 3

::: oblique.api.MySecondException
    options:
        heading_level: 3

## Data models

::: oblique.api.MyDataModel
    options:
        heading_level: 3

## Routes

::: oblique.api
    options:
        heading_level: 3
        show_root_heading: false
        show_root_toc_entry: false
        filters: ["!MyCustomException", "!MySecondException", "!MyDataModel"]
```

So, what I'm trying to do : 
* Document specific members of the module `oblique.api`, under specific headings
* Document all the other members at the end, by filtering out what was already documented in earlier headings

But when I do that, I can see the docstring of my module (`oblique.api`) at the beginning of the section `Routes`.

I'd like to exclude this module docstring, and instead simply document all remaining members. How can I do that ?

**Comment by pawamoy** [comment]
There's no other way (yet) than listing all remaining members one by one. I think it might be possible to hack something together with mkdocs-macros-plugin, adding a function in the Jinja environment that fetches data from the Python handler, and then you can use Jinja loops to iterate on members. Will try to post an example soon :slightly_smiling_face:

**Comment by astariul** [comment]
Thanks for the quick answer !
